content/zh-cn/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -324,8 +325,7 @@ static-web   1/1     Running   0               2m
 
 {{< note >}}
 <!--
-Make sure the kubelet has permission to create the mirror Pod in the API server. If not, the creation request is rejected by the API server. See
-[Pod Security admission](/docs/concepts/security/pod-security-admission) and [PodSecurityPolicy](/docs/concepts/security/pod-security-policy/).
+Make sure the kubelet has permission to create the mirror Pod in the API server. If not, the creation request is rejected by the API server. See [Pod Security admission](/docs/concepts/security/pod-security-admission) and [PodSecurityPolicy](/docs/concepts/security/pod-security-policy/).
```

Comment:
Note: It's not good to break lines if the original text didn't. But I won't revert it into single line when syncing them.